website/content/en/docs/reference/vrl/_index.md

```diff
@@ -115,6 +115,8 @@ acquainted with Vector and VRL:
 {{< jump "/docs/setup/quickstart" >}} {{< jump "/guides/level-up/transformation"
 >}}
 
+There is an online [VRL playground][vrl_playground], where you can play with VRL.
```

Comment:
```suggestion
There is an online [VRL playground][vrl_playground], where you can experiment with VRL.
```

🤔 we may also want to note that some functions are unavailable ([list](https://github.com/vectordotdev/vector/tree/master/lib/vrl/web-playground#support)) to avoid confusion?